openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -709,604 +635,510 @@ const TeamDetailsV1 = ({
-                      justify="space-between">
-                      <Col span={8}>
-                        <Searchbar
-                          removeMargin
-                          placeholder={t('label.search-entity', {
-                            entity: t('label.team'),
-                          })}
-                          searchValue={searchTerm}
-                          typingInterval={500}
-                          onSearch={handleTeamSearch}
-                        />
-                      </Col>
-                      <Col>
-                        <Space align="center">
-                          <span>
-                            <Switch
-                              checked={showDeletedTeam}
-                              data-testid="show-deleted"
-                              onClick={onShowDeletedTeamChange}
-                            />
-                            <Typography.Text className="m-l-xs">
-                              {t('label.deleted')}
-                            </Typography.Text>
-                          </span>
-
-                          {createTeamPermission && (
-                            <Button
-                              data-testid="add-team"
-                              title={
-                                createTeamPermission
-                                  ? addTeam
-                                  : t('message.no-permission-for-action')
-                              }
-                              type="primary"
-                              onClick={() => handleAddTeam(true)}>
-                              {addTeam}
-                            </Button>
-                          )}
-                        </Space>
-                      </Col>
-                      <Col span={24}>
-                        <TeamHierarchy
-                          currentTeam={currentTeam}
-                          data={childTeamList}
-                          isFetchingAllTeamAdvancedDetails={
-                            isFetchingAllTeamAdvancedDetails
-                          }
-                          onTeamExpand={onTeamExpand}
-                        />
-                      </Col>
-                    </Row>
-                  ))}
-
-                {currentTab === TeamsPageTab.USERS && (
-                  <UserTab
-                    currentPage={currentTeamUserPage}
-                    currentTeam={currentTeam}
-                    isLoading={isTeamMemberLoading}
-                    paging={teamUserPaging}
-                    permission={entityPermissions}
-                    searchText={teamUsersSearchText}
-                    users={currentTeamUsers}
-                    onAddUser={handleAddUser}
-                    onChangePaging={teamUserPagingHandler}
-                    onRemoveUser={removeUserFromTeam}
-                    onSearchUsers={handleTeamUsersSearchAction}
-                  />
-                )}
+              <>
```

Comment:
I feel Tabs component should handle this, Why we want this logic can you check it again?